generic/tclXcoalesce.c

```diff
@@ -0,0 +1,92 @@
+/*
+ * tclXcoalesce.c --
+ *
+ *  coalesce Tcl commands.
+ *-----------------------------------------------------------------------------
+ * Copyright 2017 - 2019 Karl Lehenbauer and Mark Diekhans.
+ *
+ * Permission to use, copy, modify, and distribute this software and its
+ * documentation for any purpose and without fee is hereby granted, provided
+ * that the above copyright notice appear in all copies.  Karl Lehenbauer and
+ * Mark Diekhans make no representations about the suitability of this
+ * software for any purpose.  It is provided "as is" without express or
+ * implied warranty.
+ *-----------------------------------------------------------------------------
+ */
+
+#include "tclExtdInt.h"
+
+
+/*-----------------------------------------------------------------------------
+ * TclX_CoalesceObjCmd --
+ *     Implements the TCL chmod command:
+ *     coalesce var ?var...? defaultValue
```

Comment:
This doesn't match the description of the command in the man page.  The man page says the command is:

```
coalesce ?-default value? var ?var...?
```

Reply:
Thanks here as well.  Pull request updated.